components/PageFooter.vue

```diff
@@ -1,45 +1,105 @@
 <template>
   <footer
     data-qa="footer"
-    class="p-3"
+    class="py-5 px-3"
   >
     <b-container>
       <b-row>
         <b-col
-          cols="12"
-          lg="9"
+          lg="5"
+          class="pb-4"
         >
-          <ul
-            v-if="links"
-            class="footer-link-list m-0 p-0"
-          >
-            <li
-              v-for="(footerLink, index) in links"
-              :key="index"
-            >
-              <template
-                v-if="footerLink.url"
-              >
-                <SmartLink
-                  :destination="footerLink.url"
-                  link-class="footer-link"
-                >
-                  {{ footerLink.text }}
-                </SmartLink>
-              </template>
-              <p v-else>
-                {{ footerLink.text }}
-              </p>
-            </li>
-          </ul>
+          <h5 class="text-uppercase font-weight-bold">
+            {{ $t('footer.ourMission') }}
+          </h5>
+          <p class="font-italic mb-0">
+            {{ $t('footer.ourMissionQuote') }}
+          </p>
         </b-col>
         <b-col
-          cols="12"
-          lg="3"
+          lg="4"
+          class="pb-4"
         >
-          <LangSelector
-            data-qa="language selector"
-          />
+          <b-row>
+            <b-col
+              col="6"
+              lg="12"
+            >
+              <h5 class="text-uppercase font-weight-bold">
+                {{ $t('footer.moreInfo') }}
+              </h5>
+              <ul class="footer-link-list m-0 p-0 pb-lg-5">
+                <li
+                  v-for="(footerLink, index) in footerMoreInfo"
+                  :key="index"
+                >
+                  <template
+                    v-if="footerLink.url"
+                  >
```

Comment:
Would it be possible to move the v-if to the SmartLink and get rid of the template tag?